level-1/시저-암호.py

```diff
@@ -0,0 +1,16 @@
+def solution(s, n):
+    answer = ''
+    s = list(s)
+    
+    for letter in s: 
+        #아스키코드 변환
+        a = ord(letter)
+        
+        if letter.isupper(): #대문자일 경우
+            a = ((a - 65) + n) % 26 + 65
```

Comment:
65 and 97 doesn't seem to be interpretable. Can you replace those constants?

Reply:
Changed 65 into `ord('A')` and 97 into `ord('a')`